src/api.js

```diff
@@ -11,3 +11,8 @@ export const fetchArticles = async () => {
     throw error;
   }
 };
+
+export const fetchArticleById = (article_id) => {
+  return axios.get(`${BASE_URL}/articles/${article_id}`)
+    .then((response) => response.data.article);
+};
```

Comment:
there is a way with _Axios_ to get access to a base URL without having to use a variable such as `${BASE_URL}`

[See docs](l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance) 

It's a small difference but when writing many functions it's quite a bit cleaner ✨